draft-ietf-httpbis-messaging-latest.xml

```diff
@@ -2086,7 +2092,7 @@ Connection: close
 <section title="Security Considerations" anchor="security.considerations">
 <t>
    This section is meant to inform developers, information providers, and
-   users of known security considerations relevant to HTTP message syntax
+   users about known security considerations relevant to HTTP message syntax
```

Comment:
"inform ... users of X" can be ambiguous about whether it is "users of X" that are being informed, or that users are being informed of the information X, which in this case leads to a "garden path sentence".